### DISCUSSION TITLE
Respect global gitignore?

### DISCUSSION THREAD

**Discussion by olalonde** [npm/feedback#549] [CLI] [answered]
I just noticed that a directory (`.vim/`) which is in my global gitignore file got published when I did `npm publish`. This is unexpected behavior... Shouldn't npm follow the global gitignore by default?

**Comment by MylesBorins** [comment, marked as answer]
@olalonde if you would like to make feature requests for the npm CLI the appropriate next step is to open an issue over at https://github.com/npm/rfcs
